tests/test_project/tests/test_qa.py

```diff
@@ -44,5 +44,66 @@ def test_qa_call_check_migration_name_failure(self):
                 else:
                     self.fail('SystemExit or Exception not raised')
 
+    def test_qa_call_check_commit_message_pass(self):
+        options = [
+            'commitcheck',
+            # '--token',
+            # os.environ["GH_TOKEN"],
+            '--repo', os.environ["TRAVIS_REPO_SLUG"],
+            '--message', os.environ['TRAVIS_COMMIT_MESSAGE']
+        ]
+        with patch('argparse._sys.argv', options):
+            check_commit_message()
+
+    def test_qa_call_check_commit_message_failure(self):
+        options = [
+            [
+                'commitcheck',
+                # '--token',
+                # os.environ["GH_TOKEN"],
+            ],
+            [
+                'commitcheck',
+                # '--token',
+                # os.environ["GH_TOKEN"],
+                '--message',
+                'Hello World',
+                '--repo', "ppabcd/openwisp-utils"
+            ],
+            [
+                'commitcheck',
+                # '--token',
+                # os.environ["GH_TOKEN"],
+                '--message',
+                '[qa] hello World',
+                '--repo', "ppabcd/openwisp-utils"
+            ],
+            [
+                'commitcheck',
+                # '--token',
+                # os.environ["GH_TOKEN"],
+                '--message',
+                '[qa] Hello World.',
+                '--repo', "ppabcd/openwisp-utils"
+            ],
+            [
+                'commitcheck',
+                # '--token',
+                # os.environ["GH_TOKEN"],
+                '--message',
+                '[qa] Hello World.\nTest\nFixes #20',
+                '--repo', "ppabcd/openwisp-utils"
+            ],
+            ['commitcheck']
```

Comment:
other common bad commit messages you should add to the failing tests:

```
[qa] Finished task X #2
Closes #2
```
no blank line between short desc and extended desc